addons/website_slides/views/assets.xml

```diff
@@ -29,6 +29,7 @@
                 <script type="text/javascript" src="/website_slides/static/src/js/slides_course_fullscreen_player.js"/>
                 <script type="text/javascript" src="/website_slides/static/src/js/slides_course_join.js"/>
                 <script type="text/javascript" src="/website_slides/static/src/js/slides_course_quiz.js"/>
+                <script type="text/javascript" src="/website_slides/static/src/js/slides_unsubscribe.js"/>
```

Comment:
Just some naming / namespacing convention: I would name it slides_course_unsubscribe to have "course" in it :) .

In templates name, no need to have website. prefix: for example website.slides.unsubscribe.modal could become slides.course.unsubscribe.modal . Does that seem ok for you ?